v1.0/v1.0/imported-hint.cwl

```diff
@@ -1,9 +1,6 @@
 #!/usr/bin/env cwl-runner
 cwlVersion: v1.0
 class: CommandLineTool
-hints:
```

Comment:
Is this fixed in v1.1?  Looks vaguely familiar.

Reply:
Yes, https://github.com/common-workflow-language/cwl-v1.1/blob/83bb02793b59cb039bbf12f88812678e920f727e/tests/imported-hint.cwl